fl4health/servers/base_server.py

```diff
@@ -353,14 +394,11 @@ def evaluate_round(
                     "eval_round_end": str(end_time),
                     "eval_round_time_elapsed": round((end_time - start_time).total_seconds()),
                 }
-                dummy_params = Parameters([], "None")
```

Comment:
The stored config means that we no longer have to use the strategy to configure evaluation, which was a work-around for not having access to the config in the server.